Kickstarter-iOS/Views/Controllers/RootCommentsViewControllerTests.swift

```diff
@@ -0,0 +1,31 @@
+@testable import Kickstarter_Framework
```

Comment:
Mostly curious, I've worked on projects where we separated the `Test` files into a completely new folder maybe called `ViewTests`, just an observation lol.